src/lib/converter.ts

```diff
@@ -0,0 +1,434 @@
+   * @param {Array<any>} array
+   */
+  private createRequest(item: any, index?: number, array?: any[]): void {
+    const document = this.apiParser.api as OpenAPIV3.Document
+    const { title } = document.info
+
+    const request = this.context.createRequest(
+      item.summary || item.path,
+      item.method,
+      new DynamicString(),
+      item.description,
+    )
+
+    if (item.requestBody) {
+      this.setRequesBody(
+        request,
+        item.requestBody as OpenAPIV3.RequestBodyObject,
+      )
+    }
+
+    if (item.parameters) {
+      this.setRequestParameters(item.parameters, request)
+    }
+
+    if (item.security) {
+      this.setRequestAuth(item.security, request)
+    }
+
+    if (item.responses) {
+      request.setHeader('Accepts', item.responses[0])
+    }
+
+    const requestURL = new PawURL(
+      document.paths[item.path] as OpenAPIV3.PathItemObject,
+      document,
+      item.path,
+      this.getEnviroment(),
+      request,
+    )
+
+    request.url = requestURL.fullUrl
+    this.requestGroups[item.group].appendChild(request)
+
+    if (requestURL.serverVariables) {
+      const envManager = this.envManagers[title]
+      const serverVars = requestURL.serverVariables
+
+      // Object.entries(serverVars).forEach(([varName, variable]) => {
+      //   const value = variable.default
+      //   envManager.setEnvironmentVariableValue(varName, value)
+      //   request.addVariable(
+      //     varName,
+      //     envManager.getDynamicString(varName),
+      //     variable.description || '',
+      //   )
+      // })
+    }
+  }
+
+  /**
+   * @method setRequesBody
+   * @summary
+   *
+   * @param requestBody
+   * @returns {Object<Paw.Request>}
+   */
+  private setRequesBody(
+    request: Paw.Request,
+    requestBody: OpenAPIV3.RequestBodyObject,
+  ): Paw.Request {
+    const mediaTypes = Object.keys(requestBody.content)
+
+    if (mediaTypes.length === 0) return request
+
+    const schema = requestBody.content[mediaTypes[0]]
+      .schema as OpenAPIV3.SchemaObject
+
+    request.body = JSON.stringify(jsonSchemaParser(schema, {}) as any, null, 2)
+    request.setHeader('Content-Type', mediaTypes[0])
+
+    return request
+  }
+
+  private setRequestParameters(
+    params: OpenAPIV3.ParameterObject[],
+    request: Paw.Request,
+  ): Paw.Request {
+    if (params.length === 0) return request
+    const envManager = this.getEnviroment()
+
+    function mapParameters(
+      param: ExtendedParamObject,
+      index?: number,
+      arr?: ExtendedParamObject[],
+    ): void {
+      const {
+        name,
+        description,
+        parameterExampleValue,
+        schema,
+        required,
+      } = param
+      const DYNAMIC_REQUEST_VARIABLE =
+        'com.luckymarmot.RequestVariableDynamicValue'
+
+      const variable = request.addVariable(
+        name,
+        parameterExampleValue,
+        description || '',
+      )
+
+      variable.schema = schema as any
```

Comment:
I suggest removing the `type` and `format` keys from the schema.

```suggestion
      const schema = {
        ...schema as any
      }
      delete schema.type
      delete schema.format
      variable.schema = schema
```

These two keys make Paw consider the schema as invalid. Removing those keys we have this nice popover window:

![image](https://user-images.githubusercontent.com/1319906/115698910-cd88f600-a36d-11eb-8441-d9c083ad4e99.png)